packages/docs-browser/src/dev/textResources.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
add used docs textresources here? Its kind of a documentation and overview plus will generate nice labels if in mock mode.